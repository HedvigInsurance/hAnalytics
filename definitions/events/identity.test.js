const validateEvent = require("commons/validateEvent.js");

test("validate", () => {
  const event = validateEvent(__filename, {
    memberId: "123",
  });
  expect(event).toMatchSnapshot();
});

test("validate nullable memberId", () => {
  const event = validateEvent(__filename, {
    memberId: null,
  });
  expect(event).toMatchSnapshot();
});
