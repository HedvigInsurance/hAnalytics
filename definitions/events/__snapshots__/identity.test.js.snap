// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`validate 1`] = `
Object {
  "name": "identity",
  "variables": Object {
    "member_id": "123",
  },
}
`;

exports[`validate nullable memberId 1`] = `
Object {
  "name": "identity",
  "variables": Object {
    "member_id": null,
  },
}
`;
